zookeeper-docs/src/main/resources/markdown/recipes.md

```diff
@@ -65,8 +65,7 @@ stimulate thought.
 ### Important Note About Error Handling
 
 When implementing the recipes you must handle recoverable exceptions
-(see the [FAQ](https://cwiki.apache.org/confluence/display/ZOOKEEPER/FAQ)). In
-particular, several of the recipes employ sequential ephemeral
+(see the [FAQ](https://cwiki.apache.org/confluence/display/ZOOKEEPER/FAQ)). In particular, several of the recipes employ sequential ephemeral
```

Comment:
why merge the 2 lines into 1 long line? In several other places as well. While this document has long line like this, these paragraphs are limited to shorter lines, and I think this brakes that. This is just a nit from me though.

Reply:
Breaking the lines creates grammatical errors, hence merged 2 lines into 1.